cypress/support/index.js

```diff
@@ -0,0 +1,23 @@
+// ***********************************************************
+// This example support/index.js is processed and
+// loaded automatically before your test files.
+//
+// This is a great place to put global configuration and
+// behavior that modifies Cypress.
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// For Testing Library APIs https://github.com/testing-library/cypress-testing-library
+import '@testing-library/cypress/add-commands';
```

Comment:
This files is automatically generated by Cypress. We don't need it yet, but I'm committing it in this PR because otherwise it will show up as a new file in git if folks run the tests locally.

Having said that, I added an import for Testing Library for Cypress so that folks don't need to do this explicitly in every test file.